README.md

```diff
@@ -88,57 +92,61 @@ Check if users were created successfully by accessing the identities page:
 ### Ory Cloud
 #### Taking What You Need:
 - Create an account and a project on https://console.ory.sh/login.
-- Copy the SDK Configuration url and save it.
-- Scroll down in the same page and create a Personal Access Tokens, copy the acess token that gonna show up on the bottom of your screen and save it.
-- Go to Identity Schema and click on Customize Identity Schema then change the actual schema to the code below and click Update after the change:
+- In Project settings, copy the SDK configuration URL and save it.
+- On the same page, in API keys, create a new API key, copy it, and save it.
+- Go to User Management in Identity Schema and click on Duplicate Identity Schema. Then, update the current schema with the code below and click Save after making the changes:
 ```
 {
-  "$id": "https://schemas.ory.sh/presets/kratos/identity.email.schema.json",
-  "title": "Person",
-  "type": "object",
-  "properties": {
-    "traits": {
-      "type": "object",
-      "properties": {
-        "email": {
-          "type": "string",
-          "format": "email",
-          "title": "E-Mail",
-          "ory.sh/kratos": {
-            "credentials": {
-              "password": {
-                "identifier": true
-              },
-              "webauthn": {
-                "identifier": true
-              },
-              "totp": {
-                "account_name": true
-              }
-            },
-            "recovery": {
-              "via": "email"
+    "$id": "https://schemas.ory.sh/presets/kratos/identity.email.schema.json",
```

Comment:
suggestion: I would prefer to stop putting the JSON here in the readme and reference to the file at https://github.com/AletheiaFact/aletheia/blob/stage/ory_config/identity.schema.json